<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="wheel_leg">
  <link
    name="body">
    <inertial>
      <origin
        xyz="-0.0114447581796383 -0.0008304966899257 -0.0195321753399737"
        rpy="0 0 0" />
      <mass
        value="6.37410218257267" />
      <inertia
        ixx="0.0133159702106144"
        ixy="8.46465076107667E-12"
        ixz="0.000336479893642314"
        iyy="0.0249611512851086"
        iyz="-5.82765966420402E-12"
        izz="0.0334235985838851" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="body_Lhleg">
    <inertial>
      <origin
        xyz="-0.14705 0.011637 -0.098549"
        rpy="0 0 0" />
      <mass
        value="0.52018" />
      <inertia
        ixx="0.0013773"
        ixy="-1.4931E-06"
        ixz="-0.0017679"
        iyy="0.0041441"
        iyz="-1.0002E-06"
        izz="0.0028304" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Lhleg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Lhleg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Lhleg"
    type="continuous">
    <origin
      xyz="0 0.258099999999996 0"
      rpy="0 0 0" />
    <parent
      link="body" />
    <child
      link="body_Lhleg" />
    <axis
      xyz="0 -1 0" />
  </joint>
  <link
    name="hleg_Llleg">
    <inertial>
      <origin
        xyz="0.14654 0.012275 -0.095215"
        rpy="0 0 0" />
      <mass
        value="0.33073" />
      <inertia
        ixx="0.00097779"
        ixy="2.7993E-06"
        ixz="0.0012454"
        iyy="0.0030072"
        iyz="-1.9876E-06"
        izz="0.0020673" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/hleg_Llleg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.69804 0.69804 0.69804 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/hleg_Llleg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Llleg"
    type="continuous">
    <origin
      xyz="-0.237247376845752 0.0229999999999895 -0.159000132640775"
      rpy="0 0 0" />
    <parent
      link="body_Lhleg" />
    <child
      link="hleg_Llleg" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Lleg_Lwheel">
    <inertial>
      <origin
        xyz="-4.2382E-06 0.0050966 1.157E-07"
        rpy="0 0 0" />
      <mass
        value="0.17352" />
      <inertia
        ixx="0.00024391"
        ixy="-2.062E-09"
        ixz="7.2121E-10"
        iyy="0.00046875"
        iyz="5.6291E-11"
        izz="0.00024394" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Lleg_Lwheel.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.89804 0.91765 0.92941 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Lleg_Lwheel.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Lwheel"
    type="continuous">
    <origin
      xyz="0.236786917787768 0.0280499999999997 -0.153153372684241"
      rpy="0 0 0" />
    <parent
      link="hleg_Llleg" />
    <child
      link="Lleg_Lwheel" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="body_Lrod">
    <inertial>
      <origin
        xyz="-0.159780095314809 0.011038146078772 -0.169922246812436"
        rpy="0 0 0" />
      <mass
        value="0.591101590483113" />
      <inertia
        ixx="0.00137777986284214"
        ixy="3.48484960450894E-13"
        ixz="-0.0010556297829666"
        iyy="0.00237174522976818"
        iyz="-1.43978202661194E-13"
        izz="0.00116433406366747" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Lrod.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Lrod.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Lrod"
    type="continuous">
    <origin
      xyz="-0.0923976430976348 0.2597 0.0923976430976308"
      rpy="0 0 0" />
    <parent
      link="body" />
    <child
      link="body_Lrod" />
    <axis
      xyz="0 -1 0" />
  </joint>
  <link
    name="body_Rhleg">
    <inertial>
      <origin
        xyz="0.147047845395129 0.0116371376290857 -0.0985491782989628"
        rpy="0 0 0" />
      <mass
        value="0.520183061878358" />
      <inertia
        ixx="0.0013772718483835"
        ixy="1.49309514852179E-06"
        ixz="0.00176789005413663"
        iyy="0.00414412970970827"
        iyz="-1.0002132711144E-06"
        izz="0.00283036004387134" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Rhleg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Rhleg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Rhleg"
    type="continuous">
    <origin
      xyz="0 -0.258100000000007 0"
      rpy="0 0 3.14159265358979" />
    <parent
      link="body" />
    <child
      link="body_Rhleg" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Rhleg_Rlleg">
    <inertial>
      <origin
        xyz="-0.12836687496156 0.0103192784041736 -0.0835122838421141"
        rpy="0 0 0" />
      <mass
        value="0.539924907706736" />
      <inertia
        ixx="0.00187875336625249"
        ixy="-5.37257232114557E-06"
        ixz="-0.00244221743327067"
        iyy="0.00584795629014114"
        iyz="-3.69625527021459E-06"
        izz="0.00401461678707642" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Rhleg_Rlleg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Rhleg_Rlleg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Rlleg"
    type="continuous">
    <origin
      xyz="0.237247376845051 0.0229999999999997 -0.159000132640695"
      rpy="0 0 0" />
    <parent
      link="body_Rhleg" />
    <child
      link="Rhleg_Rlleg" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Rlleg_Rwheel">
    <inertial>
      <origin
        xyz="4.23820144357379E-06 0.0050965727978543 1.15698335745318E-07"
        rpy="0 0 0" />
      <mass
        value="0.173520730578001" />
      <inertia
        ixx="0.000243908124832624"
        ixy="2.06200421139582E-09"
        ixz="-7.21212734459765E-10"
        iyy="0.000468745184860407"
        iyz="5.62906063461901E-11"
        izz="0.000243935618566337" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Rlleg_Rwheel.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/Rlleg_Rwheel.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Rwheel"
    type="continuous">
    <origin
      xyz="-0.236786917790487 0.0280499999999995 -0.153153372684284"
      rpy="0 0 0" />
    <parent
      link="Rhleg_Rlleg" />
    <child
      link="Rlleg_Rwheel" />
    <axis
      xyz="0 -1 0" />
  </joint>
  <link
    name="body_Rrod">
    <inertial>
      <origin
        xyz="0.159780095698218 0.0110381458573113 -0.169922247400722"
        rpy="0 0 0" />
      <mass
        value="0.591101597366677" />
      <inertia
        ixx="0.00137777986290563"
        ixy="-3.48484906721008E-13"
        ixz="0.00105562978268343"
        iyy="0.00237174523012257"
        iyz="-1.43978094180314E-13"
        izz="0.00116433406424617" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Rrod.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://wheel_leg/meshes/body_Rrod.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Rrod"
    type="continuous">
    <origin
      xyz="-0.0923976430976395 -0.259699999999998 0.0923976430976074"
      rpy="0 0 3.14159265358979" />
    <parent
      link="body" />
    <child
      link="body_Rrod" />
    <axis
      xyz="0 -1 0" />
  </joint>
</robot>